app/src/main/java/com/quipper/android/apollofrontpage/model/Posts.kt

```diff
@@ -0,0 +1,5 @@
+package com.quipper.android.apollofrontpage.model
+
+data class Posts(var id: Int, var title: String, var votes: Int, var author: Author)
+
+data class Author(var firstName: String, var lastName: String)
```

Comment:
[NITS] 
If these classes aren't used, we may remove :bow:

Reply:
Sorry! It is already removed in the [next PR](https://github.com/quipper/apollo-frontpage-android-app/pull/6)